core/src/main/java/com/linecorp/armeria/client/ClientOptions.java

```diff
@@ -71,8 +70,8 @@
 
     private static final ClientOptionValue<?>[] DEFAULT_OPTIONS = {
             DEFAULT_WRITE_TIMEOUT_MILLIS.newValue(DEFAULT_DEFAULT_WRITE_TIMEOUT_MILLIS),
-            DEFAULT_RESPONSE_TIMEOUT_MILLIS.newValue(DEFAULT_DEFAULT_RESPONSE_TIMEOUT_MILLIS),
-            DEFAULT_MAX_RESPONSE_LENGTH.newValue(DEFAULT_DEFAULT_MAX_RESPONSE_LENGTH),
+            DEFAULT_RESPONSE_TIMEOUT_MILLIS.newValue(Flags.defaultRequestResponseTimeoutMillis()),
+            DEFAULT_MAX_RESPONSE_LENGTH.newValue(Flags.defaultMaxRequestResponseLength()),
```

Comment:
> Is there any reason that they need to be separated?

Let's say a user writes an API server which summarizes the information retrieves from other API servers. The API server might want to limit the length of incoming request to a very small number of bytes but it might want to receive a large streaming response from other API servers.